Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
+// import CardBanner from './CardBanner'
+import CardContent from './CardContent'
+
+function CardContainer() {
+    return (
+        <div className="card-container">
+            <CardContent />
+        </div>
```

Comment:
CardContainer should be returning both CardContent and CardBanner components. I can see you have CardBanner inside CardContent, but it will create some tricky DOM strucuring and might make styling more difficult

Reply:
also remember that everything inside CardContainer should be wrapped in an `a` tag to the link mentioned in the readme